dockers/docker-database-global/global_db.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+
+import swsssdk
+import json
+globaldb = swsssdk.SonicV2Connector(host='redis_global.server', port='6385')
+globaldb.connect( globaldb.GLOBAL_APP_DB )
+fname='/etc/sonic/global_db.json'
```

Comment:
Add a comment: This is temporary and will go away eventually once all the contents of GLOBAL_APP_DB come from each FSI.